circleci/README.md

```diff
@@ -0,0 +1,36 @@
+# Circle CI
+ 
+### Install
+ 
+Running the below command to install the whole CircleCI command set.
+ 
+```
+/nc csm_install circleci
+```
+ 
+### Prerequisite:
+To be able to run the commands below, you need to create 3 secrets.
+ 
+These are :
+`projectToken` - CircleCI allows you to create a Token on a project basis to access API, Make sure you provide the token here which can access all those branches which you want to use via this command.
+ 
+This secret should be created with the following naming convention. `projectName_token`. Where the project name is your circleCi project name. example `webApp_token`.
```

Comment:
This isn’t required though - so this is a tip? I think once you have the api token what it’s actually called in the circle dashboard isn’t important.

Reply:
Yes, What it’s actually called in the circle dashboard isn’t important but it's easy for user to remember the name, and based on the name we can send the specific token in the request.

The reason behind `projectName_token` is, let's say we have 2 projects in CCI `commander` and `main`.
So, I have created 2 API access tokens 1 for each.

Now, I want to run a workflow for `commander` first and then for `main`. This, I would have to pass as a command parameter, and based on that program can check for the token as all the secrets are available to all the commands present inside same app.

/nc circleci_workflow_run commander develop  -> commander_token
/nc circleci_workflow_run main develop -> main_token